blockchain/api.go

```diff
@@ -69,8 +69,8 @@ type ProfileRegistryAPI interface {
 type EventsAPI interface {
 	GetEvents(ctx context.Context, filter *EventFilter) (chan *Event, error)
 	GetLastBlock(ctx context.Context) (uint64, error)
-	GetMarketFilter(fromBlock *big.Int) *EventFilter
-	GetMultiSigFilter(addresses []common.Address, fromBlock *big.Int) *EventFilter
+	GetMarketFilter(fromBlock *big.Int, blocksBatchSize int64) *EventFilter
```

Comment:
Looks like it's not the parametere of call, but configuration parameter of EventsAPI

Reply:
ok will do